tests/integ/sagemaker/jumpstart/conftest.py

```diff
@@ -113,6 +133,35 @@ def _teardown():
     bucket = s3_resource.Bucket(test_cache_bucket)
     bucket.objects.filter(Prefix=test_suite_id + "/").delete()
 
+    # delete private hubs
+    _delete_hubs(sagemaker_session)
+
+
+def _delete_hubs(sagemaker_session):
+    # list Hubs created by PySDK integration tests
+    list_hub_response = sagemaker_session.list_hubs(name_contains=HUB_NAME_PREFIX)
+
+    for hub in list_hub_response["HubSummaries"]:
+        if hub["HubName"] != SM_JUMPSTART_PUBLIC_HUB_NAME:
+            # delete all hub contents first
+            _delete_hub_contents(sagemaker_session, hub["HubName"])
+            sagemaker_session.delete_hub(hub["HubName"])
+
+
+def _delete_hub_contents(sagemaker_session, test_hub_name):
```

Comment:
nit: this deletes only the model references (not models or other contents), recommend to take in a optional arg of which content type to delete